src/components/Conference.tsx

```diff
@@ -71,6 +72,8 @@ export interface IConferenceState {
     // TODO(yunsi): Currently we just store this information,
     // but we need to add UI to show microphone activity for remote stream based on this.state.remoteIsSpeaking.
     remoteIsSpeaking: { [id: string]: boolean };
+    // TODO(yunsi): Define type for audioMonitor
+    audioMonitor: any;
```

Comment:
Define a class that wraps the Hark implementation and use the type here

Reply:
Done